data/mods/MindOverMatter/powers/learning_eocs/overall.json

```
@@ -1,19 +1,81 @@
 [
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_PSI_STUDYING_POWER",
+    "effect": [
+      { "math": [ "u_psi_studying_tick_experience_counter", "+=", "contemplation_factor(1)" ] },
+      {
+        "run_eocs": [
+          {
+            "id": "EOC_PSI_STUDYING_POWER_INTEGER_TICK_OVER",
+            "condition": { "math": [ "u_psi_studying_tick_experience_counter", ">=", "1" ] },
+            "effect": [
+              { "math": [ "u_spell_exp(u_latest_studied_power_name)", "+=", "1" ] },
+              { "math": [ "u_psi_studying_tick_experience_counter", "=", "0" ] }
```

I haven't done the math, but it might be kinder not to destroy the overflow?
```suggestion
              { "math": [ "u_psi_studying_tick_experience_counter", "-=", "1" ] }
```

---

I think with = 0 it would be about 4% slower, which would add up over time.